docker-storage-setup.sh

```diff
@@ -435,11 +438,11 @@ setup_lvm_thin_pool () {
   tpool=`get_configured_thin_pool`
 
   if [ -n "$tpool" ]; then
-     local escaped_pool_lv_name=`echo $POOL_LV_NAME | sed 's/-/--/g'`
+     local escaped_pool_lv_name=`echo $CONTAINER_THINPOOL | sed 's/-/--/g'`
```

Comment:
Also, instead of hardcoding based on /etc/sysconfig/docker-storage-setup as inputfile, can we use pool name as docker-lv when user has not specified CONTAINER_THINPOOL and we are not creating new thing pool.

IOW, if we are creating new thin pool we make sure COINTAINER_THINPOOL is specified, otherwise if CONTAINER_THINPOOL is not specified, then we fall back to using docker-pool.